docs/src/tutorials/implementing.md

```diff
@@ -713,7 +713,7 @@ Solver-specific attributes should be specified by creating an appropriate
 subtype of [`AbstractModelAttribute`](@ref), [`AbstractOptimizerAttribute`](@ref),
 [`AbstractVariableAttribute`](@ref), or [`AbstractConstraintAttribute`](@ref).
 
-For example, Gurobi.jl adds attributes for multiobjective optimization by
+For example, Gurobi.jl adds attributes for multi-objective optimization by
```

Comment:
Not sure about this one, @xgandibleux seems to use `multiobjective` in vOptSolver, vOptGeneric, ...

Reply:
Both are used but multiobjective, multicriteria, nondominated are prefered to multi-objective, multi-criteria, non-dominated.

Reply:
Thanks for confirming